phpBB/includes/acp/acp_prune.php

```diff
@@ -397,7 +397,7 @@ function get_prune_users(&$user_ids, &$usernames)
 			$joined_after = request_var('joined_after', '');
 			$active = request_var('active', '');
 
-			$count = request_var('count', 0);
+			$count = ($request->variable('count', '') === '') ? false : $request->variable('count', 0);
```

Comment:
could also try `$request->variable('count', -1)` ? and then check for `($count >= 0)`